card/mod/03_machines/spec/set/type/scss_spec.rb

```diff
@@ -37,7 +37,7 @@
     end
   end
 
-  it "should process links and inclusions but not urls" do
+  it "should process links and nests but not urls" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.